libgalois/include/katana/PropertyGraph.h

```diff
@@ -135,6 +144,49 @@ class KATANA_EXPORT GraphTopology {
 
   bool empty() const noexcept { return num_nodes() == 0; }
 
+  /**
+   * @param node node to get degree for
+   * @returns Degree of node N
+   */
+  size_t degree(Node node) const noexcept {
+    auto edge_range = edges(node);
+    return std::distance(edge_range.begin(), edge_range.end());
+  }
+
+  Edge original_edge_id(const Edge& eid) const noexcept { return eid; }
+
+  Node original_node_id(const Node& nid) const noexcept { return nid; }
+
+#if 0
```

Comment:
Either remove or implement these before merge.

Reply:
Will do.

Reply:
Leaning towards removing these as the `local_ranges` are not well defined for NUMAArray (esp. with Interleaved allocation policy).

Reply:
remove local_ranges.